docs/documentation/fxFlex-API.md

```diff
@@ -182,12 +184,12 @@ Developers should consult the **[Known Issues](http://caniuse.com/#feat=flexbox)
 [Resources](http://caniuse.com/#feat=flexbox) sections.
 
 <a href="http://caniuse.com/#feat=flexbox" target="_blank">
-![caniuseflexbox](https://cloud.githubusercontent.com/assets/210413/21288118/917e3faa-c440-11e6-9b08-28aff590c7ae.png)
+  <img alt="caniuseflexbox" src="https://cloud.githubusercontent.com/assets/210413/21288118/917e3faa-c440-11e6-9b08-28aff590c7ae.png">
```

Comment:
This should be `[![caniuse](https://cloud.githubusercontent.com/assets/210413/21288118/917e3faa-c440-11e6-9b08-28aff590c7ae.png)](http://caniuse.com/#feat=flexbox)` with no `<a>` tags